cookbook/deployment/tools.rst

```diff
@@ -62,12 +63,35 @@ manually taking other steps (see `Common Post-Deployment Tasks`_).
 Using Build Scripts and other Tools
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-There are also high-quality tools to help ease the pain of deployment. There
-are even a few tools which have been specifically tailored to the requirements of
-Symfony, and which take special care to ensure that everything before, during,
-and after a deployment has gone correctly.
+There are also tools to help ease the pain of deployment. Some of them have been
+specifically tailored to the requirements of Symfony.
+
+`Capifony`_
+    Provides a specialized set of tools on top of `Capistrano`_, tailored
+    specifically to Symfony projects.
+
+`sf2debpkg`_
+    Helps you build a native Debian package for your Symfony project.
+
+`Magallanes`_
```

Comment:
Thanks for your suggestion: Fabric is indeed very popular, so I've added a reference to it in 9e01941